include/simgrid/host.h

```diff
@@ -26,6 +27,8 @@ XBT_PUBLIC(void*) sg_host_extension_get(sg_host_t host, size_t rank);
 XBT_PUBLIC(sg_host_t) sg_host_by_name(const char *name);
 XBT_PUBLIC(const char*) sg_host_get_name(sg_host_t host);
 XBT_PUBLIC(xbt_dynar_t) sg_hosts_as_dynar();
+XBT_PUBLIC(xbt_swag_t) sg_host_get_process_list(sg_host_t host);
```

Comment:
Yes, you need that function building a dynar. Or even better if your caller code is in C++, build a std::vector.